about.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>Global school about page</title>
+ <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
+ <link rel="stylesheet" href="css/mobile_style.css">
+ <link rel="stylesheet" href="css/Destop_style.css">
+ <link rel="stylesheet" href="css/about_style.css">
+</head>
+<body>
+ <header>
```

Comment:
- [ ] In both desktop size pages make sure 
1. It is divided into a top dark bar and a bottom white bar.
2. The top short bar section with social media links, logout, and language links aligned to the right.
3. The taller bottom bar with a logo and a list of page links.
4. The top bar contains a logo to the left.
1. The top bar contains a list of page links aligned to the right.
Take a look at the original design below 👇 
![Screenshot (179)](https://user-images.githubusercontent.com/67087664/153582844-26fc5664-5fad-4d8a-8358-23ea42a26545.png)